src/js/ia/Helpers.js

```diff
@@ -30,7 +30,7 @@
         var timestring = full? " days ago" : "d";
         if (date) {
             // expected date format: YYYY-MM-DDTHH:mm:ssZ e.g. 2011-04-22T13:33:48Z
-            date = date.replace("/T.*Z/", "");
+            date = date.replace(/T.*Z/, "");
```

Comment:
@jdorweiler Yes I was thinking that too! I think this is a relic of _before_ we used momentjs. I could change it to that instead if we're not in a hurry